### ISSUE TITLE
AvgPool3d --> padding？

### ISSUE THREAD

**Issue by eli5168** [jiecaoyu/pytorch_imagenet#1]
It seems Pytorch do not support padding in AvgPool3d.

**Comment by jiecaoyu** [comment]
Hi, I once had the same problem. You need to upgrade the PyTorch. The padding is supported in the newest version ([link](http://pytorch.org/docs/master/nn.html#torch.nn.AvgPool3d)).

**Comment by eli5168** [comment]
Thanks
